src/scss/_vars.scss

```diff
@@ -153,7 +153,7 @@ $font-monospace: "Open Sans Condensed", "Roboto Mono", monospace;
 // ==== FOCUS STATE =====
 $focus-glow: 0 0 4px rgba($blue-700, 0.5);
 $focus-glow-disabled: $focus-glow;
-$focus-glow-invalid: 0 0 4px rgba($red-status-900, 0.5);
+$focus-glow-invalid: 0 0 4px rgba($red-status-500, 0.5);
```

Comment:
This looks good to me. However, just make sure design includes this in their docs.
The focus states were left out of the checkbox states and missed. I think they were missed for radios and some other components as well.

Reply:
@danielmdesigns, can you provide design recommendations for `required` and `invalid` states.  These states could occur during validation phase (for example, form validation).

Reply:
@ssalinas24 @danielmdesigns we received confirmation from design to keep `required` and `invalid` states.  I updated these states to match our v2.0 Checkbox component styled specifications.